src/main/webapp/join.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Plant Battles</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
+    <script src="script.js"></script>
+    <script src="https://trefle.io/api/plants?q=rosemary&token=VU4zZmx5SnF3dmxVYTVsRlBTNGlqZz09"></script>
```

Comment:
u can take this out if we aren't using trefle anymore

Reply:
oh, yep I missed that. Thanks!